cmd/traffic_ctl/host.cc

```diff
@@ -0,0 +1,98 @@
+/** @file
+
+  host.cc
+
+  @section license License
+
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+#include "traffic_ctl.h"
+#include <P_RecUtils.h>
+const std::string stat_prefix = "host_status.";
+static int
+status_get(unsigned argc, const char **argv)
+{
+  if (!CtrlProcessArguments(argc, argv, nullptr, 0) || n_file_arguments < 1) {
+    return CtrlCommandUsage("host status_get HOST  [HOST  ...]", nullptr, 0);
+  }
+
+  for (unsigned i = 0; i < n_file_arguments; ++i) {
+    CtrlMgmtRecord record;
+    TSMgmtError error;
+    std::string str = stat_prefix + file_arguments[i];
+
+    error = record.fetch(str.c_str());
+    if (error != TS_ERR_OKAY) {
+      CtrlMgmtError(error, "failed to fetch %s", file_arguments[i]);
+      return CTRL_EX_ERROR;
+    }
+
+    if (REC_TYPE_IS_STAT(record.rclass())) {
+      printf("%s %s\n", record.name(), CtrlMgmtRecordValue(record).c_str());
+    }
+  }
+
+  return CTRL_EX_OK;
+}
+
+static int
+status_down(unsigned argc, const char **argv)
+{
+  if (!CtrlProcessArguments(argc, argv, nullptr, 0) || n_file_arguments < 1) {
+    return CtrlCommandUsage("host status_set HOST  value", nullptr, 0);
```

Comment:
Should this be `status_set` and not `status_down`?

Reply:
good catch, i'll fix the usage message to say status_down.